net/mwan3-luci/files/usr/lib/lua/luci/controller/mwan3.lua

```diff
@@ -65,7 +65,7 @@ end
 
 function getInterfaceStatus(ruleNumber, interfaceName)
 	if ut.trim(sys.exec("uci -p /var/state get mwan3." .. interfaceName .. ".enabled")) == "1" then
-		if ut.trim(sys.exec(ip .. "route list table " .. ruleNumber)) ~= "" then
+		if ut.trim(sys.exec("cat /var/run/mwan3/iface_state/" .. interfaceName .. " 2>/dev/null" )) == "online" then
```

Comment:
Add `local fs = require "nixio.fs"` to the top and change this line to `if ut.trim(fs.readfile("/var/run/mwan3/iface_state/%s" % interfaceName)) == "online" then`

Reply:
thank you for the hint will update the PR next week